Oleksii_Fedorchuk/HW18_Oleksii_Fedorchuk/human.py

```diff
@@ -0,0 +1,42 @@
+class Human:
+    def __init__(self, name: str, age: int, gender: str):
+        self.__name = name
+        self.__age = age
+        self.__gender = gender
+        self.__status = "alive"
+        self.__age_limit = 100
+
+    def grow(self):
+        self.__is_alive()
```

Comment:
this functionality is not tested. No test for trying grow dead person

Reply:
Will be fixed in an update